chronos/apps/http/httpclient.nim

```diff
@@ -1079,13 +1211,16 @@ proc open*(request: HttpClientRequestRef): Future[HttpBodyWriter] {.
   try:
     let headers = request.prepareRequest()
     request.connection.state = HttpClientConnectionState.RequestHeadersSending
+    request.setTimestamp()
```

Comment:
shouldn't this be part of `prepareRequest`?

Reply:
`prepareRequest` is not related to actual data sending process, so why we need to start from there?